test/functional/ui/cmdline2_spec.lua

```diff
@@ -0,0 +1,39 @@
+local n = require('test.functional.testnvim')()
```

Comment:
Before I forget, let's add a note like this here and in messages2_spec.lua.

```suggestion
-- Tests for the ui2 (protocol-driven UI), which will replace the legacy UI.

local n = require('test.functional.testnvim')()
```